signup.js

```diff
@@ -0,0 +1,16 @@
+const passWord = document.querySelectorAll("#password");
```

Comment:
❗️ 수정요청
password는 한단어니 아래처럼 작성해주세요.

```suggestion
const password = document.querySelectorAll("#password");
```

추가로 변수명은 어떤 값인지 파악할 수 있는 가장 중요한 단서입니다~ password라는 변수명은 password 타입의 input을 잘 나타내지 못하고 있습니다.
이런 경우 passwordInput 같은 식으로 네이밍 하시는 것을 추천드려요!